tests/test_multiprocessing.py

```diff
@@ -209,7 +219,7 @@ def test_process_spawn(spawn_context):
 
     logger.add(writer, context=spawn_context, format="{message}", enqueue=True, catch=False)
 
-    process = spawn_context.Process(target=subworker, args=(logger,))
+    process = spawn_context.Process(target=subworker_spawn, args=(logger,))
```

Comment:
I would prefer we do not touch the existing unit tests. Could you please create a new `tests/test_reinstall.py` file where we'll put all tests we need?